hms-api/src/handler/hackathon/create.ts

```diff
@@ -15,6 +16,10 @@ export async function create(event, context, callback) {
       request.endDate,
     );
 
+    if (hackathon) {
+      await createCategory('General', 'General', hackathon.id);
+    }
```

Comment:
createHackathon should never return nothing, checking here is a bit redundant.  Also it would be good to leave a comment here as to why we create a `General` category

Reply:
I would also say that this should be done in the service, and not in the controller.  The controller should pretty dumb and not need to know that the hackathon requires this default category.  I would also argue that the Hackathon is invalid until this category exists so it would be best if it doesn't return from the service until it does

Reply:
so... I would say move this call into the service.  And then for the tests for the service you will need to mock this call by "spy"ing on it and then also ensure it is called with the correct values.